plugins/aea-ledger-hwi/aea_ledger_hwi/__init__.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2023 Valory AG
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+
+"""Python package wrapping the public and private key cryptography and support for hardware wallet interactions."""
+
+from .hwi import (  # noqa: F401
```

Comment:
why not absolute path?

Reply:
addressed https://github.com/valory-xyz/open-aea/pull/583